pandas/formats/style.py

```diff
@@ -461,9 +463,22 @@ def apply(self, func, axis=0, subset=None, **kwargs):
 
         Notes
         -----
+        The output shape of ``func`` should match the input, i.e. if
+        ``x`` is the input row, column, or table (depending on ``axis``),
+        then ``func(x.shape) == x.shape`` should be true.
+
```

Comment:
I'll push commit tonight with this (very easy, two line change). I had written a test that violated the rule, but "worked" 😳 . Just need to write additional tests.